coordinatorlib/src/main/scala/com/socrata/datacoordinator/secondary/sql/SqlSecondaryManifest.scala

```diff
@@ -396,9 +397,12 @@ class SqlSecondaryManifest(conn: Connection) extends SecondaryManifest {
       try {
         stmt.executeUpdate()
       } catch {
+        case PostgresUniqueViolation(_*) =>
+          conn.rollback(savepoint)
+          throw ResyncLaterSecondaryException("wait for another resync")
         case ex: SQLException =>
           conn.rollback(savepoint)
-          throw ex
+          throw ResyncLaterSecondaryException(ex.getMessage)
       }
```

Comment:
If we don't catch other errors, it will be closer to the original behavior.  Isn't that desirable?